tests/NLog.UnitTests/ConfigFileLocatorTests.cs

```diff
@@ -48,9 +45,234 @@ namespace NLog.UnitTests
     using NLog.Config;
     using NLog.Layouts;
     using NLog.LayoutRenderers;
+    using NLog.UnitTests.Mocks;
 
     public class ConfigFileLocatorTests : NLogTestBase, IDisposable
     {
+        private readonly string _tempDirectory;
+
+        public ConfigFileLocatorTests()
+        {
+            _tempDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString("N"));
+            Directory.CreateDirectory(_tempDirectory);
+        }
+
+        void IDisposable.Dispose()
+        {
+            if (Directory.Exists(_tempDirectory))
+                Directory.Delete(_tempDirectory, true);
+        }
+
+        [Fact]
+        void FuncLayoutRendererRegisterTest1()
+        {
+            LayoutRenderer.Register("the-answer", (info) => "42");
+            Layout l = "${the-answer}";
+            var result = l.Render(LogEventInfo.CreateNullEvent());
+            Assert.Equal("42", result);
+        }
+
+        [Fact]
+        void FuncLayoutRendererRegisterTest1WithXML()
+        {
+            LayoutRenderer.Register("the-answer", (info) => "42");
```

Comment:
Just me moving unit tests out of the `#if NETSTANDARD `-condition because of the exe-file-compile-logic. Next step will probably be to remove the exe-file-compile and only use mocks. But like to see that my other changes is not affecting the old tests before changing the tests also.